src/envs/behavior_options.py

```diff
@@ -1123,11 +1088,9 @@ def place_ontop_obj_pos(
               " to find plan to continuous params" + f" {place_rel_pos}")
         return None
 
-    if not ret_option_model:
-        print(f"PRIMITIVE: placeOnTop {obj.name} success! Plan found with " +
-              f"continuous params {place_rel_pos}. Executing now.")
-        return create_place_policy(plan)
-
+    original_orientation = list(
```

Comment:
if env gets changed within `place_obj_plan`, I guess it's possible that this is not the "original" orientation with respect to `state`. is that matter / does that matter?

Reply:
good question! Two things:
1. `place_obj_plan` resets the state at the end, so we should guaranteeably be back to the robot state consistent with `state`
2. Barring the above, we only use `original_orientation` for navigation; it's vestigial for grasping and placing

Reply:
cool cool. merge when ready!